dec2/cli/daskd.py

```diff
@@ -14,10 +15,15 @@
               show_default=True,
               required=False,
               help="Filepath to the instances metadata")
+@click.option("--nprocs",
+              default=1,
+              show_default=True,
+              required=False,
+              help="Number of worker processes")
```

Comment:
I suggest a different help string, "Number of processes per worker", just to make it clear that this doesn't change the number of worker nodes.